docs/en/reference/sql/dql/LIMIT_CLAUSE.md

```diff
@@ -16,11 +16,12 @@ SELECT ... LIMIT ...
 ```
 
 ## Description
-For the standalone version, `LIMIT` is supported in all conditions. For the cluster version, the execution modes, which support this clause, are shown below.
+
+Limit clause is not supported in online preview mode. By practise, limit clause should used together with order by clause, to constrains the result rows into a unique order. OpenMLDB do not support order by clause yet, it is exepcted to get inconsistent results for the same query.
```

Comment:
`Limit` clause is not supported in online request mode. By practice, `limit` clause should be used together with `order by` clause, to constrain the result rows in a unique order. OpenMLDB does not support `order by `clause yet; without it it may get inconsistent results for the same query.


some typos fixed

Reply:
Thx. 

But
- online preview mode is what the request mode called in the docs elsewhere
- `to constrains the result rows into a unique order` is picked up from psql: https://www.postgresql.org/docs/15/queries-limit.html. Though should be fine too